src/components/DocxDownloadTemplates/DocxDownloadTemplate.tsx

```diff
@@ -0,0 +1,86 @@
+import {
+  Data,
+  BlockOneLevelDeep,
+  LectureTwoLevelsDeep,
+  CourseThreeLevelsDeep,
+  DownloadableContent,
+} from '../../types'
+import Markdown from '../Markdown/Markdown'
+import Abstract from './Abstract'
+import Authors from './Authors'
+import CiteAs from './CiteAs'
+import Duration from './Duration'
+import Heading from './Heading'
+import LearningOutcomes from './LearningOutcomes'
+import Level from './Level'
+import References from './References'
+
+export const isBlockOneLevelDeep = (
+  data: Data<DownloadableContent>
+): data is Data<BlockOneLevelDeep> =>
+  !!(data as Data<BlockOneLevelDeep>).attributes.Document
+export const isLectureTwoLevelsDeep = (
+  data: Data<DownloadableContent>
+): data is Data<LectureTwoLevelsDeep> =>
+  !!(data as Data<LectureTwoLevelsDeep>).attributes.Blocks
+export const isCourseThreeLevelsDeep = (
+  data: Data<DownloadableContent>
+): data is Data<CourseThreeLevelsDeep> =>
+  !!(data as Data<CourseThreeLevelsDeep>).attributes.Lectures &&
+  !!(data as Data<CourseThreeLevelsDeep>).attributes.CourseCreators
```

Comment:
👆🏻 Hehe fair enough

Reply:
It's not an ideal solution for sure. Appending an enum type to the data after API calls seemed like a good solution, but it's also risky in case we create additional API calls and forget to append the types it could become a headache.